### ISSUE TITLE
Sexlab Interactive Arousal doesn't work with OSL Aroused

### ISSUE THREAD

**Issue by ozooma10** [ozooma10/OSLAroused#15]
TL;DR: Sexlab Interactive Arousal doesn't work with OSL Aroused. But it might could be fixed by copying over sla_internal quest and slaInternalScr script. It could fix other potential backward compatibility issues as well.

So I was using SLIA, and seem to discover nothing seems to work, even though I already forwarded all SLIA changes to vanilla items.
Further investigation reveals that SLIA need [quest: sla_internal] to work properly. However OSL's sexlab aroused esm is missing it.
In the end I fixed this by copying slaInternalScr script from our good ol' Sexlab Arousal Redux, then created an sla_internal quest record identical to what it is from SL arousal redux. Then everything starts working again. However now OSL's SL arousal esm requires sexlab.esm as a master.

**Comment by ozooma10** [comment]
fixed with #12
